printer-creality-cr10v3-bltouch.cfg

```diff
@@ -0,0 +1,229 @@
+#Stock Creality CR-10 V3 with BL-Touch and runout filament sensor
+[stepper_x]
+step_pin: PF0 #ar54
+dir_pin: PF1 #ar55
+enable_pin: !PD7 #!ar38
+microsteps: 16
+rotation_distance: 40 
+#Step distance is deprecated and will be removed in future releases
+#step_distance: .0125 <=> 0.0125 mm * 200 steps *16 microsteps = 40 mm
+endstop_pin: ^PE5 #^ar3
+position_endstop: 0
+position_max: 300
+homing_speed: 50
+
+[stepper_y]
+step_pin: PF6 #ar60
+dir_pin: PF7 #ar61
+enable_pin: !PF2 #!ar56
+microsteps: 16
+rotation_distance: 40
+endstop_pin: ^PJ1 #^ar14
+position_endstop: 0
+position_max: 300
+homing_speed: 50
+
+[stepper_z]
+step_pin: PL3 #ar46
+dir_pin: !PL1 #!ar48
+enable_pin: !PK0 #!ar62
+microsteps: 16
+rotation_distance: 8
+#Step distance is deprecated and will be removed in future releases
+#step_distance: .0025 <=> 0.0025 mm * 200 steps * 200 steps/rev = 8 mm 
+position_max: 400
+position_min: -4
+endstop_pin: probe:z_virtual_endstop
+
+[safe_z_home]
+home_xy_position: 104.25,147.6 # Change coordinates to the center of your print bed relative to the probe
+speed: 80
+z_hop: 10               
+z_hop_speed: 10
+
+[extruder]
+max_extrude_only_distance: 150.0
+max_extrude_cross_section: 1.00 #For start G-Code that was set-up in ideaMaker
+step_pin: PA4 # ar26
+dir_pin: !PA6 # !ar28
+enable_pin: !PA2 # !ar24
+microsteps: 16
+rotation_distance: 7.7201944 # 16 microsteps * 200 steps/rotation / steps/mm
+#Correction formula is new_rotation_distance = old_rotation_distance * mmsExtracted / 100.0
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+min_extrude_temp: 166 # 166C is the minimum extrude temperature, this allows to print at 170C, given possible temp fluctuation with PID algorithm 
```

Comment:
What purpose does this comment serve?
`min_extrude_temp` is a safety setting to avoid damaging the extrusion system when trying to print cold

Reply:
The min. extrude temp was set to that values, because I once printed a PLA temp tower that had a minimum temperature of 170ºC and as such it would cause a Klipper error, due to the PID algorithm stabilization procedure reaching a temperature slightly below 170ºC. I will remove this configuration line, since it was a leftover from a print test.